packages/block-library/src/buttons/edit.native.js

```diff
@@ -24,20 +25,43 @@ import styles from './editor.scss';
 const ALLOWED_BLOCKS = [ buttonBlockName ];
 const BUTTONS_TEMPLATE = [ [ 'core/button' ] ];
 
-function ButtonsEdit( {
-	isSelected,
-	attributes,
-	onDelete,
-	onAddNextButton,
-	shouldDelete,
-	isInnerButtonSelected,
-} ) {
-	const { align } = attributes;
+function ButtonsEdit( { attributes: { align }, clientId, isSelected } ) {
 	const [ resizeObserver, sizes ] = useResizeObserver();
 	const [ maxWidth, setMaxWidth ] = useState( 0 );
 	const shouldRenderFooterAppender = isSelected || isInnerButtonSelected;
 	const { marginLeft: spacing } = styles.spacing;
 
+	const { getBlockOrder, isInnerButtonSelected, shouldDelete } = useSelect(
+		( select ) => {
+			const {
+				getBlockCount,
+				getBlockOrder: _getBlockOrder,
```

Comment:
You can keep the name `getBlockOrder` here and have
```
return {
	getBlockOrder,
```
a few lines below

Reply:
Actually, our linting rules wouldn't allow this due to duplication/shadowing of variable names. The underscore pattern used here is already used in a few places in our codebase, hence why I did it here.